docs/10.EXT-REFERENCE-HANDLER.md

```diff
@@ -1,3 +1,329 @@
+# Common methods to handle properties
+
+## jerryx_property_entry
+
+**Summary**
+
+Structure to define an array of methods with `name` and `function` fields which
+can be registered to a target object. When setting the `function` value
+a native C method must be used.
```

Comment:
The description is outdated, since not only external functions can be registered now.